waltid-services/waltid-issuer-api/build.gradle.kts

```diff
@@ -236,6 +236,9 @@ ktor {
         )
     }
     jib {
+        container {
+            workingDirectory = "/waltid-issuer-api"
```

Comment:
could you mention why this is required
if I recall correctly, yesterday there was a fix where these paths were removed
is it a different thing? don't the other workflows need it too, e.g. compose-testing?

Reply:
perhaps in the pr description